src/gc/impl/conservative/gc.d

```diff
@@ -2634,6 +2770,14 @@ struct Gcx
             ConservativeGC._inFinalizer = false;
         }
 
+        // minimize() should be called only after a call to fullcollect
+        // terminates with a sweep
+        if (doMinimize || lowMem)
```

Comment:
Can `doMinimize` be converted to a function parameter instead of a field? It's unlikely to actually find completely unused pool, so it's probably not too bad if it is skipped sometimes.

Reply:
I don;t think it can because it needs to be set when:
* lowMem
* tryAllocNewPool fails

we could avoid minimizing after a failed tryAllocNewPool  and call minimize() only on lowMem (at least for the forking GC only)

Reply:
Maybe it just needs a better name, e.g. `minimizeAfterNextCollection`.